src/interactive-config.coffee

```diff
@@ -103,7 +93,7 @@ interactiveConfig.prompt = (config = {}, callback) ->
     when: (answers) -> (! fs.existsSync('circle.yml') && ! fs.existsSync('.travis.yml'))
   }
 
-  inquirer.prompt questions, callback
+  inquirer.prompt(questions).then(callback)
```

Comment:
We discussed this internally, but for the record, it's inquirer's strange API: https://github.com/sboudrias/Inquirer.js#installation